SU-SWS/stanford_components
# changed style to better acommondate incorrect image style usage

# READY FOR REVIEW

# Summary
- Added styles to prevent the incorrect image size used.
- Ordered /patterns page items alphabetically by label.

# Needed By (Date)
- August?

# Urgency
- low

# Steps to Test

1. checkout branch
2. change the image previews to any other image (different aspect ratio, size etc)
3. clear cache
4. verify filmstrip & tall filmstrip still looks normal.

# Associated Issues and/or People
- https://github.com/SU-SWS/stanford_paragraph_types/pull/57

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
